// Copyright 2021 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";

option java_multiple_files = true;
option java_package = "org.chromium.chrome.browser.sharing.proto";

package chrome_browser_sharing;

// Required in Chrome.
option optimize_for = LITE_RUNTIME;

// Message used to deliver push notifications through optimization guide.
// Next tag: 2
message OptimizationGuidePushNotification {
  // This is a HintNotificationPayload.
  bytes hint_notification_payload_bytes = 1;
}
